Tasks/powerShell/task.loc.json

```diff
@@ -30,7 +34,7 @@
       "label": "ms-resource:loc.input.label.scriptName",
       "defaultValue": "",
       "required": true,
-      "helpMarkDown": "Relative path from repo root of the PowerShell script file to run."
+      "helpMarkDown": "Path of the script to execute relative to the default working directory of the agent."
```

Comment:
Path of script to execute.  Should be fully qualified path or relative to the default working directory.